src/adapterManager.js

```diff
@@ -380,7 +380,7 @@ adapterManager.callBids = (adUnits, bidRequests, addBidResponse, doneCb, request
     return;
   }
 
-  let [clientBidRequests, serverBidRequests] = bidRequests.reduce((partitions, bidRequest) => {
+  let [clientBidderReqs, serverBidRequests] = bidRequests.reduce((partitions, bidRequest) => {
```

Comment:
Is there a reason we're changing this variable? I find the new name less readable. I'd prefer clientBidderRequests, personally.